.idea/sonarlint/issuestore/index.pb

```diff
@@ -0,0 +1,27 @@
+
+K
+res/css/elements/images.css,a/a/aa1eb33bfe5737e1634389f156f53ab134c734b4
+I
+res/css/elements/main.css,4/5/45d27eae8b06a02c486528b1455e542d6ad7b756
+K
+res/css/elements/tables.css,a/3/a33bc8aa2cff919e9a7119e46bf9980684482bf7
+@
+res/css/vars.css,3/1/317b059f9fa9180e4fe00e91001876cdd9441ac7
+@
+res/js/search.js,d/7/d7f083556ad9587f949b91cf0688d62c70eba763
+@
+res/js/events.js,5/7/57bae627b14706f3e9db2dc667a5aa630a5a976d
+H
+res/css/elements/nav.css,4/b/4bf9c060a6d40a79ac1b02120040f15fcd122a78
+O
+res/css/elements/link-lists.css,1/7/17aef164b5806ec01236a03751f04e24f230c61e
+K
+res/css/elements/header.css,f/8/f89cf31dcca546209d6a548c8634ac4c521a5211
+U
+%res/css/elements/general-elements.css,f/b/fb8770ebf9224b8a4a4298d8189b0f5357627e75
+K
+res/css/elements/footer.css,4/8/48e1d5777023f544a8e70c1c4cc8989d276bab6e
+9
+	README.md,8/e/8ec9a00bfd09b3190ac6b22251dbb1aa95a0579d
+=
+
.editorconfig,6/2/626a747c8694eaae66f2afb3d1a745b045782c70
```

Comment:
do not add :-)

Reply:
Ähm ... Ich arbeite dran. IDE's tun IDE-Dinge :D